frontend/app/settings/tax/page.tsx

```
@@ -29,6 +29,8 @@ import MutationButton, { MutationStatusButton } from "@/components/MutationButto
 import { Dialog, DialogContent, DialogFooter, DialogHeader, DialogTitle } from "@/components/ui/dialog";
 import { linkClasses } from "@/components/Link";
 import { Label } from "@/components/ui/label";
+import { SettingsTaxSkeleton } from "@/components/SettingsSkeleton";
+import { Suspense } from "react";
```

ESLint

[error] '/home/jailuser/git/node_modules/react/index.js' imported multiple times.

(import/no-duplicates)